stdlib/Makefile

```diff
@@ -0,0 +1,34 @@
+default: get
+
+PKG3_VERSION := f4139648bfb4cdde34e50947a73f71c7ee8d29a5
+PKG3_URL := https://github.com/vchuravy/Pkg3.jl
+
+Pkg3:
+	git clone $(PKG3_URL) $@
+Pkg3/$(PKG3_VERSION).commit: Pkg3
+	git -C $< checkout $(PKG3_VERSION)
+	-rm $</*.commit
+	touch $@
+checkout-pkg3: Pkg3/$(PKG3_VERSION).commit
+get-pkg3:  checkout-pkg3
+clean-pkg3:
+	rm -rf Pkg3
+
+SHA_VERSION := 8f2c9bb537097b661ac91580d18d20af51e5af22
+SHA_URL := https://github.com/vchuravy/SHA.jl
```

Comment:
Same question here

Reply:
No this is just my fork that drops the Compat requirement